src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -1224,10 +1224,15 @@ private boolean loadAlias(final Resource resource, Map<String, Map<String, Colle
             containingResource = resource;
         }
 
+        if (containingResource == null) {
+            log.warn("containingResource is null for alias on {}, skipping.", resource.getPath());
+            return false;
+        }
+ 
         final Resource parent = containingResource.getParent();
 
         if (parent == null) {
-            log.debug("parent is null for alias on {}.", resource.getName());
+            log.warn("parent is null for alias on {}, skipping.", resource.getPath());
```

Comment:
The message is not entirely accurate, as ``parent`` could be also the grand-parent of ``resource`` :-)

Reply:
you mean exactly *which*? I'd pefer to do that separately as part of a separate ticket/PR.